### PR TITLE
Refactor raspberrypi specific semi tests

### PR DESCRIPTION
**Pull request by horia-delicoti**

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -444,17 +445,23 @@ if (options.interactiveTests) {
...
+          'Logging into the device (ssh to the host OS)',
+          'Set "console=serial0,115200" in "/mnt/boot/cmdline.txt" so that the kernel outputs logs to serial',
+          'Plug in the USB serial cable into device, like this "https://elinux.org/RPi_Serial_Connection"',
           'Make sure there are no "Device Configuration" variables configured'
         ],
         do: [
-          'Run `minicom -b 115200 -o -D /dev/tty***`, replacing the tty device accordingly to your host',
-          'Set `RESIN_HOST_CONFIG_dtoverlay=pi3-miniuart-bt` as a "Device Configuration" variable'
+          `Run "minicom -b 115200 -o -D ${options.serialConnection}"`,
+          'Add custom variable "RESIN_HOST_CONFIG_dtoverlay = pi3-miniuart-bt" as a "Device Configuration" variable'
```
Comment (jviotti) [inline lib/main.js]: Should this `pi3-` variable be set even when the device is a Raspberry Pi 1?

File: lib/main.js [hunk 2]
```diff
@@ -464,19 +471,25 @@ if (options.interactiveTests) {
     ava.test(`${options.deviceType}: Enable serial port on UART1/MiniUART/ttyS0`, (test) => {
       return utils.runManualTestCase(test, {
         prepare: [
-          'Set "console=serial0,115200" in /mnt/boot/cmdline.txt so that the kernel outputs logs to serial',
+          'Logging into the device (ssh to the host OS)',
```
Comment (jviotti) [inline lib/main.js]: Should you add a teardown step to close the host OS SSH session?

File: lib/main.js [hunk 3]
```diff
@@ -444,17 +445,23 @@ if (options.interactiveTests) {
     ava.test(`${options.deviceType}: Enable serial port on UART0/ttyAMA0`, (test) => {
       return utils.runManualTestCase(test, {
         prepare: [
-          'Set "console=serial0,115200" in /mnt/boot/cmdline.txt so that the kernel outputs logs to serial',
+          'Logging into the device (ssh to the host OS)',
```
Comment (jviotti) [inline lib/main.js]: We probably need to tell the user to close the host OS SSH session as well

**Comment by telphan** [comment]
No longer relevant.

**Pull request closed without merging**